src/info/repo.rs

```diff
@@ -224,53 +218,39 @@ impl Repo {
     }
 
     pub fn get_name_and_url(&self) -> Result<(String, String)> {
```

Comment:
Should the `return Ok(Default::default())` be changed to return `Err`s?

Reply:
I think this would break onefetch as it aborts the information retrieval process for repository information entirely. If there was an improvement to make, I'd certainly go for `Result<Option<(…)>>` here.

Reply:
Got it, I'm OK with the existing code, then :smiley: :+1: